TF_Settings_Web/src/Pages/Camera/Masking/MaskingSlider.tsx

```diff
@@ -36,31 +35,22 @@ export const MaskingSlider: React.FC<MaskingSliderProps> = ({ direction, masking
     // ===== UseEffects =====
     useEffect(() => {
         if (!sliderRef.current) return;
-        const top = Number.parseFloat(getComputedStyle(sliderRef.current).top);
-        const left = Number.parseFloat(getComputedStyle(sliderRef.current).left);
+        const size = innerHeight < innerWidth ? innerHeight * 0.45 : innerWidth * 0.96 - innerHeight * 0.1;
+        const top = direction === 'lower' ? size - 5 : 0;
+        const left = direction === 'right' ? size - 5 : 0;
 
-        setPosition(direction, maskingValue, canvasInfo.size, { top, left }, sliderRef.current);
+        setPosition(direction, maskingValue, canvasSize, { top, left }, sliderRef.current);
         setInitialSliderPos({ top, left });
-    }, []);
+    }, [innerHeight]);
```

Comment:
💬 innerWidth here as well?

Reply:
Updated to use the `isLandscape` hook instead